src/pe/options.rs

```diff
@@ -37,6 +39,7 @@ impl ParseOptions {
         Self {
             resolve_rva: false,
             parse_attribute_certificates: false,
+            parse_tls_data: false,
```

Comment:
just curious, what is the rationale for skipping this for te?

Reply:
you are right, it might be better to leave it enabled